api/core/model_runtime/model_providers/siliconflow/llm/deepseek-v2-chat.yaml

```diff
@@ -1,11 +1,9 @@
-model: deepseek-ai/deepseek-v2-chat
+model: deepseek-ai/DeepSeek-V2-Chat
```

Comment:
If you change the model id directly, it will cause the user model currently in use to disappear, so it is recommended that you only change the label value.

Reply:
Thanks for your advice.
	1.	Currently, the new model is named deepseek-ai/DeepSeek-V2-Chat (the same as on [Huggingface](https://huggingface.co/deepseek-ai/DeepSeek-V2-Chat)). Therefore, deepseek-ai/deepseek-v2-chat is deprecated. We want to rename the model in Dify, but we support existing users in continuing to use the deepseek-ai/deepseek-v2-chat label.
	2.	As @takatost mentioned, changing the name from deepseek-ai/deepseek-v2-chat to deepseek-ai/DeepSeek-V2-Chat may corrupt existing workflows. How can I alias the model name without affecting the current online business?
![image](https://github.com/user-attachments/assets/b51423a9-82db-4114-b5ec-98789ddf6dd8)

Reply:
You can create a new yaml config, and the old config set `deprecated: true`